be/src/runtime/mysql_table_writer.cpp

```diff
@@ -133,7 +133,7 @@ Status MysqlTableWriter::_build_viewers(Columns& columns) {
 Status MysqlTableWriter::_build_insert_sql(int from, int to, std::string_view* sql) {
     _stmt_buffer.clear();
     int num_cols = _viewers.size();
-    fmt::format_to(_stmt_buffer, "INSERT INTO {} VALUES ", _mysql_tbl);
+    fmt::format_to(std::back_inserter(_stmt_buffer), "INSERT INTO {} VALUES ", _mysql_tbl);
```

Comment:
The `std::back_inserter` seems no necessary. Because the `_stmt_buffer` have been `clear()` before calling the `fmt::format_to`

Reply:
As I understand, `std::back_inserter()` or other output iterator is necessary in `fmt::format_to()` in version 8 whether `_stmt_buffer` is empty or not. It only accepts OutputIt as the first argument in version 8 and `fmt::format_to(_stmt_buffer,` is not allowed any more.

Reply:
OK. I get it.

Reply:
Thanks for the review.